ds-collector/ds-collector

```diff
@@ -791,6 +792,8 @@ discover_jmx() {
   # nodetoolCredentials="-Dcom.sun.jndi.rmiURLParsing=legacy"
   if [[ -n ${jmxUsername} ]] && [[ -n ${jmxPassword} ]] ; then
     nodetoolCredentials="-u $jmxUsername -pw $jmxPassword"
+  elif [[ -n ${jmxUsername} ]] && [[ -n ${jmxPasswordFilePath} ]] ; then
+    nodetoolCredentials="-u $jmxUsername -pwf $jmxPasswordFilePath"
```

Comment:
this works for nodetool executions, but there are other places jmxPassword is used that will now not work (both in this script and in the rust file `ds-collector/rust-commands/collect-info.rs`).

so i think we still have to read and set `jmxPassword`
e.g.
```suggestion
  while IFS= read -r line; do
      # Skip empty lines
      [[ -z "$line" ]] && continue
      # Read first two fields from the line
      read -r _role _password <<< "$line"
      if [[ "${_role}" == "${jmxUsername}" ]]; then
          jmxPassword="${_password}"
          break
      fi
    done < "${jmxPasswordFilePath}"
    nodetoolCredentials="-u $jmxUsername -pwf $jmxPasswordFilePath"
```